lib/action.js

```diff
@@ -0,0 +1,76 @@
+const request = require('./request');
+const { sign } = require('./signing');
+const ClientError = require('./error');
+
+// This doesn't really make sense anymore as the version won't change (0.0.0-development)
```

Comment:
Is this something we want to change now? Is there a ticket for this?

Reply:
In my opinion, if we don't want to deal with this now we should remove it. This data now doesn't make sense. I assume that we want to keep this functionality. But if we don't want to deal with this now, we should create an issue to track it. @shuisman what do you think?

Reply:
This is not needed anymore, so we can remove it!